src/accountlist/accountlistReducers.js

```diff
@@ -0,0 +1,57 @@
+import { fromJS } from 'immutable';
+
+import {
+  REALM_ADD,
+  LOGIN_SUCCESS,
+  LOGOUT,
+  ACCOUNT_REMOVE,
+} from '../account/accountActions';
+
+const initialState = fromJS([]);
+
+export default (state = initialState, action) => {
```

Comment:
I am making the most recent account bubble up for UI purposes too - most recent is on top.
If it wasn't for that, it would have not been natural to do so, just to indicate which one is active.